src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -637,14 +637,17 @@
 
 helps['functionapp identity assign'] = """
 type: command
-short-summary: assign or disable managed service identity to the web app
+short-summary: assign managed service identity to the web app
```

Comment:
user cannot disable MSI ?

Reply:
`az functionapp/webapp identity remove` disables all managed identities on an app.

Reply:
got it

Reply:
Before this update, `identity assign` makes an app:
whatever identities are assigend -> only system managed identity assigned